cli/src/declarations.ts

```diff
@@ -268,11 +289,22 @@ export interface CapacitorConfig {
      *
      * Overrides global `hideLogs` option.
      *
-     * @since 1.1.0
+     * @since 2.1.0
```

Comment:
```suggestion
     * @since 1.1.0
```
For iOS it was implemented on 1.1.0, since Android allows to hide logs natively, but some people requested the same configuration option for Android and was added in 2.1.0

Reply:
Ah, I didn't know the history and assumed it was a typo.